pkg/autoscaler/scaling/autoscaler_test.go

```diff
@@ -545,10 +546,26 @@ func newTestAutoscalerWithScalingMetric(t *testing.T, targetValue, targetBurstCa
 		metrics, pc, deciderSpec, ctx), pc
 }
 
+// equateInt32 equates int32s with given path with ±-1 tolerance.
+// This is needed due to rounding errors across various platforms.
+func equateInt32(field string) cmp.Option {
+	eqOpt := cmp.Options{
+		cmp.FilterValues(
+			func(a, b int32) bool { return true },
+			cmp.Comparer(func(a, b int32) bool {
```

Comment:
Can this Comparer be passed directly to FilterPath below or is the FilterValues needed?

Reply:
I think we can. We don't filter specific values...